File: backend/src/main/java/com/onlinemenu/domain/account/openingAccount/OpeningAccountResponse.java
package com.onlinemenu.domain.account.openingAccount;

import lombok.Data;

@Data
public class OpeningAccountResponse {

    private Boolean success;

    private String description;

    private String password;

    public OpeningAccountResponse(Boolean success, String description, String password) {
        this.success = success;
        this.description = description;
        this.password = password;
    }
}

File: frontend/src/ui/components/loginPage/register/openingAccountResponse/OpeningAccountResponse.js
import React from 'react';
import {Alert} from 'react-bootstrap';
import styles from './openingAccountResponse.css';

const OpeningAccountResponse = props => (
  <section className={styles.section}>
    <Alert bsStyle={props.openingAccountResponse.success ? 'success' : 'danger'}>
      <h4>{props.openingAccountResponse.description}</h4>
      <Choose>
        <When condition={props.openingAccountResponse.success}>
          <h4>Password: {props.openingAccountResponse.password}</h4>
        </When>
      </Choose>
    </Alert>
  </section>
);

export default OpeningAccountResponse;
